src/main/java/ua/com/juja/microservices/keepers/slackbot/model/SlackParsedCommand.java

```diff
@@ -1,19 +1,79 @@
 package ua.com.juja.microservices.keepers.slackbot.model;
 
-import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ua.com.juja.microservices.keepers.slackbot.exceptions.WrongCommandFormatException;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
- * @author
+ * @author Konstantin Sergey
  */
 public class SlackParsedCommand {
-   private String from;
-   private String text;
-   private Map<String, UserDTO> users;
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    private final String SLACK_NAME_PATTERN = "@([a-zA-z0-9\\.\\_\\-]){1,21}";
+    private String from;
+    private String text;
+    private Map<String, UserDTO> users;
+    private List<String> slackNames;
+    private int userCount;
 
     public SlackParsedCommand(String from, String text, Map<String, UserDTO> users) {
+        if (!from.startsWith("@")) {
+            logger.debug("add '@' to slack name [{}]", from);
+            from = "@" + from;
+        }
         this.from = from;
         this.text = text;
         this.users = users;
-        //todo task #11
+        slackNames = receiveSlackNames();
+        userCount = slackNames.size();
```

Comment:
add logger.debug here with text "SlackParsedCommand created. fromSlackName: {} text: {} slackNames in the text {slackNames.toString} users: {users.toString}"
or you can do better